server/src/main/java/com/objectcomputing/checkins/services/member_skill/MemberSkillController.java

```diff
@@ -56,25 +58,16 @@ public MemberSkillController(MemberSkillServices memberSkillServices,
                 .body(error);
     }
 
-    @Error(exception = MemberSkillBadArgException.class)
-    public HttpResponse<?> handleBadArgs(HttpRequest<?> request, MemberSkillBadArgException e) {
-        JsonError error = new JsonError(e.getMessage())
-                .link(Link.SELF, Link.of(request.getUri()));
-
-        return HttpResponse.<JsonError>badRequest()
-                .body(error);
-    }
-
-    @Error(exception = MemberSkillAlreadyExistsException.class)
-    public HttpResponse<?> handleAlreadyExists(HttpRequest<?> request, MemberSkillAlreadyExistsException e) {
+    @Error(exception = AlreadyExistsException.class)
+    public HttpResponse<?> handleAlreadyExists(HttpRequest<?> request, AlreadyExistsException e) {
         JsonError error = new JsonError(e.getMessage())
                 .link(Link.SELF, Link.of(request.getUri()));
 
         return HttpResponse.<JsonError>status(HttpStatus.CONFLICT).body(error);
     }
 
-    @Error(exception = MemberSkillNotFoundException.class)
-    public HttpResponse<?> handleNotFound(HttpRequest<?> request, MemberSkillNotFoundException e) {
+    @Error(exception = NotFoundException.class)
+    public HttpResponse<?> handleNotFound(HttpRequest<?> request, NotFoundException e) {
```

Comment:
Don't think we need this. Should be handled by the global handler NotFoundHandler in the exceptions/handlers folder

Reply:
@moshirim Holly is correct. This handler can be removed.

Reply:
The same is true for the AlreadyExistsException handler, as Holly mentioned.